app/person.rb

```diff
@@ -0,0 +1,42 @@
+class Person
+  def initialize(age, name: 'Unknown', parent_permission: true)
```

Comment:
- [ ] Please from the project requirement, you should set _Getters for `@id`, `@name`, and `@age`_. and 
_Setters for `@name` and `@age`_.  the `:name` and `:age` instance variables are the only ones that should be able to be modified I recommend that they should be defined as attr_accessors (`attr_accessor :name, :age`) and the `:id` which should be able to be retrieved only so the Getter should be defined (`attr_reader :id`)